Description: Fix FTBFS on mipsel* due to custom library search paths.
 This patch removes library search -L/usr/lib and include paths
 defined by upstream. Since all the libraries and headers needed are
 available in default path these flags are redundant also since
 /usr/lib may have different purpose on various architecture like
 mips* it causes FTBFS. Thanks to YunQiang Su for bringing this up.
Author: Vasudev Kamath <kamathvasudev@gmail.com>
Last-Updated: 2013-09-15
Bug-Debian: http://bugs.debian.org/722765

--- a/config.mk
+++ b/config.mk
@@ -7,21 +7,17 @@
 PREFIX = /usr/local
 MANPREFIX = ${PREFIX}/share/man
 
-X11INC = /usr/X11R6/include
-X11LIB = /usr/X11R6/lib
-
 # Xinerama
-XINERAMALIBS = -L${X11LIB} -lXinerama
+XINERAMALIBS =  -lXinerama
 XINERAMAFLAGS = -DXINERAMA
 
 # includes and libs
-INCS = -I. -I/usr/include -I${X11INC}
-LIBS = -L/usr/lib -lc -L${X11LIB} -lX11 ${XINERAMALIBS}
+LIBS = -lc -lX11 ${XINERAMALIBS}
 
 # flags
 CPPFLAGS += -DVERSION=\"${VERSION}\" ${XINERAMAFLAGS}
 #CFLAGS = -g -std=c99 -pedantic -Wall -O0 ${INCS} ${CPPFLAGS}
-CFLAGS += -std=c99 -pedantic -Wall ${INCS} ${CPPFLAGS}
+CFLAGS += -std=c99 -pedantic -Wall ${CPPFLAGS}
 #LDFLAGS = -g ${LIBS}
 LDFLAGS += -s ${LIBS}
 
